### PR TITLE
Fix osc address

### PR DESCRIPTION
**Pull request by fand**
In old ZIG SIM, every OSC message had prefix `/ZIGSIM` in the address... 😇 

## Changes
- Add `/ZIGSIM` prefix to OSC addresses
- Add `Service.osc` to make OSC messages

### CODE REVIEW

File: ZIGSIMPlus/Utils.swift [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 import UIKit
+import SwiftOSC
```
Comment (nozomuplus) [inline ZIGSIMPlus/Utils.swift]: Is this necessary?

File: ZIGSIMPlus/Models/Services/Service.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import SwiftOSC
+import SwiftyJSON
+
+protocol Service {
+    func toLog() -> [String]
+    func toOSC() -> [OSCMessage]
+    func toJSON() throws -> JSON
+}
+
+extension Service {
```
Comment (nozomuplus) [inline ZIGSIMPlus/Models/Services/Service.swift]: Good!

File: ZIGSIMPlus/Utils.swift (hunk 1 above)
Comment (fand) [reply ZIGSIMPlus/Utils.swift]: ah nope, I just forgot to remove this line.
fixed in bc11b8190

**Comment by fand** [comment]
Thanks!!

**Pull request merged**